pyserini/vectorizer/_base.py

```diff
@@ -115,7 +127,7 @@ def get_vectors(self, docids: List[str]):
                 matrix_data.append(tfidf)
 
         vectors = csr_matrix((matrix_data, (matrix_row, matrix_col)), shape=(num_docs, self.vocabulary_size))
-        return normalize(vectors, norm='l2')
```

Comment:
how about `norm=l2` as an optional parameter so it doesn't break downstream code that depend on this, e.g., 20newsgroup classification?

Then caller can set `norm=None` to turn off?